src/core/i18n/translate.js

```diff
@@ -49,6 +44,12 @@ export default function translate(options = {}) {
 
     Translate.displayName = `Translate[${getDisplayName(WrappedComponent)}]`;
 
-    return Translate;
+    function mapStateToProps({ api, i18n }) {
+      return {
+        i18n: makeI18n(i18n, api.lang),
```

Comment:
Also, I need to better understand when redux decides to render a `connect()`'d component or not. Their docs are not helpful.